Source/WebCore/html/track/VTTCue.cpp

```diff
@@ -210,16 +210,16 @@ void VTTCueBox::applyCSSProperties()
     setInlineStyleProperty(CSSPropertyWritingMode, cue->getCSSWritingMode());
 
     // the 'top' property must be set to top
-    std::visit(WTF::makeVisitor([&] (double top) {
+    std::visit(WTF::makeVisitor([this, protectedThis = Ref { *this }] (double top) {
```

Comment:
I reported these unnecessary protectors to @rniwa. He's going to look into the static analysis to see if we can do better.